internal/server/routes_test.go

```diff
@@ -108,30 +108,6 @@ func TestBindsEmptyRequest(t *testing.T) {
 	assert.NilError(t, err)
 }
 
-func TestGetRoute(t *testing.T) {
```

Comment:
I think this test was probably useful before, but now it does not really do much. We have good test coverage for all of our real production routes, so we don't need to test an artificial route.